examples/async_client_hello_callback/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "async_client_hello_callback"
+version = "0.1.0"
+authors = ["AWS s2n"]
+edition = "2021"
+
+[dependencies]
+s2n-quic = { version = "1", path = "../../quic/s2n-quic", features = ["provider-tls-s2n", "unstable_client_hello"]}
+tokio = { version = "1", features = ["full"] }
+s2n-tls = { version = "=0.0.21", features = ["quic"] }
```

Comment:
Is it necessary to pin to a specific s2n-tls version?

Reply:
yea, in the example we probably shouldnt.